File: greendao-code-modifier/test-files/relation-input/Order.java
package io.objectbox.codemodifier.test;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Generated;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.Relation;

import java.util.List;

@Entity
public class Order {

    @Id
    long id;

    long customerId;

    @Relation()
    Customer customer;

}

File: objectbox-gradle-plugin/test-gradle-projects/kotlin-android/src/main/kotlin/io/objectbox/test/kotlin/Order.kt
package io.objectbox.test.kotlin

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import io.objectbox.annotation.NameInDb
import io.objectbox.relation.ToOne
import java.util.*

@Entity
@NameInDb("ORDERS")
class Order {

    @Id(assignable = true)
    var id: Long = 0
    var date: Date? = null
    var customerId: Long = 0
    var text: String? = null

    lateinit var customer: ToOne<Customer>

    lateinit var customerWithoutIdProperty: ToOne<Customer>

}
